src/components/tasks/card/index.tsx

```diff
@@ -41,31 +41,57 @@ const Card: FC<Props> = ({
     const timeLeft = localEndsOn.valueOf() - currentDate.valueOf();
     return !statusNotOverDueList.includes(cardDetails.status) && timeLeft <= 0;
   }
-
+  
   function stripHtml(html: string) {
     const tmp = document.createElement('DIV');
     tmp.innerHTML = html;
     return tmp.textContent || tmp.innerText || '';
   }
-
+  
   function handleChange(event: any, changedProperty: keyof typeof cardDetails) {
     if (event.key === 'Enter') {
       const toChange: any = cardDetails;
       toChange[changedProperty] = stripHtml(event.target.innerHTML);
-
+      
```

Comment:
There are some extra spaces green one added , not a major blocker though but this might create merge issues to others

Reply:
I don't why it is showing it as a change, I checked the develop branch this space already exists there